images/manageiq-base/Dockerfile

```diff
@@ -45,46 +45,65 @@ LABEL name="manageiq-base" \
-      chrony            \
-      cmake             \
-      cronie            \
-      file              \
-      gcc-c++           \
-      git               \
-      glibc-langpack-en \
-      hostname          \
-      http-parser       \
-      initscripts       \
-      iproute           \
-      libcurl-devel     \
-      libpq             \
-      libtool           \
-      libxml2-devel     \
-      libxslt-devel     \
-      logrotate         \
-      lvm2              \
-      make              \
-      net-tools         \
-      nmap-ncat         \
-      nodejs            \
-      openldap-clients  \
-      openscap-scanner  \
-      openssl-devel     \
-      patch             \
-      psmisc            \
-      postgresql-devel  \
-      redhat-rpm-config \
-      ruby-devel        \
-      sqlite-devel      \
-      wget              \
-      which             \
-      xz                \
-      &&                \
+      ansible                     \
+      ansible-runner              \
+      cmake                       \
+      cockpit-ws                  \
+      cockpit-ssh                 \
+      cyrus-sasl                  \
+      cyrus-sasl-plain            \
+      file                        \
+      freeipmi                    \
+      gcc-c++                     \
+      git                         \
+      glibc-langpack-en           \
+      hostname                    \
+      http-parser                 \
+      ipmitool                    \
+      iproute                     \
+      libcurl-devel               \
+      libpq                       \
+      libssh2-devel               \
+      libtool                     \
+      libxml2-devel               \
+      libxslt-devel               \
+      logrotate                   \
+      lvm2                        \
+      make                        \
+      net-snmp                    \
+      net-snmp-libs               \
+      net-snmp-utils              \
+      nmap-ncat                   \
+      nodejs                      \
+      openldap-clients            \
+      openscap-scanner            \
+      openssl-devel               \
+      OpenIPMI                    \
+      patch                       \
+      postfix                     \
+      postgresql-devel            \
+      procps-ng                   \
+      python3-devel               \
+      qpid-proton-c-devel         \
+      redhat-rpm-config           \
+      ruby-devel                  \
+      socat                       \
+      sqlite-devel                \
+      v2v-conversion-host-ansible \
+      wget                        \
+      wmi                         \
+      xz                          \
+      &&                          \
     dnf clean all
 
+# Install python packages the same way the appliance does
+RUN curl -L https://raw.githubusercontent.com/ManageIQ/manageiq-appliance-build/master/kickstarts/partials/post/python_modules.ks.erb | bash -
```

Comment:
`master` needs to be replaced with appropriate branch as we cut branches. Not sure if it makes more sense to use ARG...  If not, one more place for @Fryguy to change.

Also, do we need to support other ARGs (GIT_AUTH, _ORG, etc.) here?

Reply:
Hm, possibly. Right now we don't actually pull anything from appliance-build so this is net-new.

I guess our options are to either treat it the same as the other repos including ARGs, AUTH, etc. Or just copy the content like we do for the packages. What do you think?